tox.ini

```diff
@@ -34,6 +34,7 @@ basepython =
     python310: python3.10
     python311: python3.11
     python312: python3.12
+    python3.11
```

Comment:
Only 3.11? This seems very weird.

Reply:
It's the fallback version for when no other version is found for the environment. Uv downloads and installs it. Do you want to run linting and coverage with 3 different interpreters each time?

How did it behave prior to the uv PR?

Reply:
I *think* coverage ran within the current env, not something by tox?

Reply:
I have changed it to just `python3` now, which should select any Python 3 dist (system or uv-managed) that is supported by the project.